infrastructure/cli/cmd/create/create.go

```diff
@@ -4,25 +4,51 @@ import (
 	"fmt"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"k8s.io/client-go/util/homedir"
+	"log"
+	"path/filepath"
 )
 
 var (
-	provider  string
-	CreateCmd = &cobra.Command{
+	provider   string
+	kubeConfig string
+	version    string
+	CreateCmd  = &cobra.Command{
 		Use:   "create",
 		Short: "Creates an instance of Airy Core",
 		Long:  ``,
 		Run:   create,
 	}
 )
 
+const defaultKubeConfig = "$HOME/.kube/config"
+
 func init() {
+	if home := homedir.HomeDir(); home != "" {
+		CreateCmd.Flags().StringVar(&kubeConfig, "kubeconfig", defaultKubeConfig, "(optional) absolute path to the kubeconfig file")
+	} else {
+		CreateCmd.Flags().StringVar(&kubeConfig, "kubeconfig", "", "absolute path to the kubeconfig file")
+	}
+
 	CreateCmd.Flags().StringVar(&provider, "provider", "", "One of the supported providers (aws|local). Default is aws")
+
 	viper.SetDefault("provider", "aws")
 }
 
 func create(cmd *cobra.Command, args []string) {
 	fmt.Println("⚙️  Creating core with provider", provider)
+
+	if kubeConfig == defaultKubeConfig {
+		kubeConfig = filepath.Join(homedir.HomeDir(), ".kube", "config")
+	}
+
+	helm := New(kubeConfig, "develop", "default")
+	if err := helm.Setup(); err != nil {
+		log.Fatalf("setting up Helm failed with err %v", err)
```

Comment:
we don't use log anywhere as it comes with its own format and right now we don't have an opinion about it. So Printf plus exit is how we're doing this (not sure we don't have any log left... I did remove the few we had a few weeks ago but things move fast :))